third_party/windows-setup.py

```diff
@@ -0,0 +1,21 @@
+import os
+import sys
+import subprocess
+
+current_path = os.path.dirname(os.path.realpath(__file__))
+
+# cleanup
+os.chdir("C:\Python34\Lib\site-packages")
+subprocess.call("rm -rf misoc*")
+subprocess.call("rm -rf migen*")
+subprocess.call("rm -rf lite*")
+
+# pull
+for d in ["migen", "misoc", "liteeth", "litescope", "fx2lib"]:
+	os.chdir(os.path.join(current_path, d))
+	subprocess.call("git pull")
```

Comment:
I'm unsure the `git pull` is right here. Will that pull to the version that the submodule is pointed at.